stanford_paragraph_types.features.filter.inc

```diff
@@ -0,0 +1,176 @@
+<?php
+/**
+ * @file
+ * stanford_paragraph_types.features.filter.inc
+ */
+
+/**
+ * Implements hook_filter_default_formats().
+ */
+function stanford_paragraph_types_filter_default_formats() {
```

Comment:
Can you move this filter format to it's own submodule in this repo and mark it as a dependency. I foresee this filter moving in to our core set.